content/python/concepts/PIP/pip.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: ''
```

Comment:
According to the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md), `Title` is missing which caused the test to fail.

Reply:
Excuse me! I left that there when troubleshooting. I will fill in these fields and pass on what I get in response

Reply:
Ah, let me try to help you pass in. I am opening it in my editor.